bin/run.sh

```diff
@@ -1,36 +1,26 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 if [ $# -lt 3 ]
 then
     echo "Usage:"
-    echo "./bin/run.sh two_fer ~/input/ ~/output/"
+    echo "./bin/run.sh two-fer ~/input/ ~/output/"
     exit 1
 fi
 
 problem_slug="$1"
 input_folder="$2"
 output_folder="$3"
-compiler_error_log="compiler.log"
+tmp_folder="/tmp/solution"
 
-cp -r $input_folder /opt/test-runner/lib/$problem_slug
-cd /opt/test-runner/lib
+mkdir -p $output_folder
 
-find . -mindepth 1 -type f | grep 'Test.java' | xargs -I file sed -i "s/@Ignore(.*)//g;s/@Ignore//g;" file
-cat <<EOF >settings.gradle
-include '${problem_slug}'
-EOF
+rm -rf $tmp_folder
+mkdir -p $tmp_folder
 
-gradle --offline addListeners compileJava compileTestJava 2> /tmp/$compiler_error_log
+cd $tmp_folder
+cp -R $input_folder/* .
 
-cd /opt/test-runner/bin
+find . -mindepth 1 -type f | grep 'Test.java' | xargs -I file sed -i "s/@Ignore(.*)//g;s/@Ignore//g;" file
 
-if [ -s "/tmp/$compiler_error_log" ]
-then 
-    echo "Compiler error, processing"
-    ./build_compilation_error_results.sh /tmp/$compiler_error_log > $output_folder/results.json
-else
-    echo "Running tests"
-    cd /opt/test-runner/lib
-    gradle --offline addListeners test
-    mv results.json $output_folder
-fi
+java -jar /opt/test-runner/autotest-runner.jar $problem_slug
+mv results.json $output_folder
```

Comment:
One future optimization is to pass in the output directory to the test runner and immediately write to the output directory

Reply:
Yep, for now let's just get this working and then we can worry about an optimization like that.